dependencies/jessie/foreman-tasks-core/changelog

```diff
@@ -0,0 +1,5 @@
+ruby-foreman-tasks-core (0.1.1-1) stable; urgency=low
```

Comment:
I don't like the name of the package, but given the current naming conventions, we don't have a choice :(